modules/social_features/social_profile/social_profile.module

```diff
@@ -290,6 +291,29 @@ function social_profile_preprocess_form_element_label(&$variables) {
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function social_profile_preprocess_field__field_profile_image(array &$variables): void {
+  if ($variables['element']['#view_mode'] === 'compact') {
```

Comment:
This code was in a template (the "profile--compact.html.twig" file) of the theme before but in this case, we can not modify link attributes so I've just removed hardcoded markup from that template and made the same here. Since it was in the theme so I'd say that this code should be as is now.

Reply:
Could you add explanation to the code?

Reply:
Done.